### ISSUE TITLE
Responsive stagePadding

### ISSUE THREAD

**Issue by aleebraccia** [OwlCarousel2/OwlCarousel2#1967]
Is it possible to modify/resize stagePadding every time that window is resized?

**Comment by martyfmelb** [comment]
Yes, just set your `stagePadding` option to a function which returns a number.
